ui/src/app/subnets/urls.ts

```diff
@@ -30,6 +34,18 @@ const getVLANLink = (id?: VLAN["id"]): string | null =>
   isId(id) ? generateNewURL(`/vlan/${id}`) : null;
 const getSubnetLink = (id?: Subnet["id"]): string | null =>
   isId(id) ? generateNewURL(`/subnet/${id}`) : null;
+const getNetworksLocation = ({
```

Comment:
As far as I can see you can `history.replace` with the query params as part of the path string, in which case you  could just reference `urls.index` instead and then this function would not be required.